spec/factories/card.rb

```diff
@@ -3,6 +3,6 @@
     original_text "How are you?"
     translated_text "Kamusta?"
     review_date DateTime.now.getutc
-    user
+    user_id 1
```

Comment:
@davetoxa i'm getting a `Validation failed: Email has already been taken` error. i think it initializes a new user object every time the factory is called. not too sure because i don't fully understand this 'magical part'. i just made it retrieve a user_id and the tests are now fixed